egs2/mucs21_subtask2/asr1/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+lang=hi-en
+
+train_set="hi-en/train/transcripts"
+valid_set="hi-en/test/transcripts"
+test_set="hi-en/test/transcripts"
+
+asr_config=conf/train.yaml
+lm_config_=conf/lm.yaml
+
+./asr.sh \
+    --lang $lang \
+    --ngpu 2 \
+    --expdir exp \
+    --local_data_opts $lang \
+    --use_lm true \
+    --skip_upload false \
```

Comment:
This should leave it as default (remove this line)
The main user would not upload the model.